python/lsst/ip/isr/isrTask.py

```diff
@@ -610,8 +621,10 @@ def saturationDetection(self, exposure, amp):
         """
         if not math.isnan(amp.getSaturation()):
             maskedImage = exposure.getMaskedImage()
-            dataView = maskedImage.Factory(maskedImage, amp.getRawBBox())
-            isr.makeThresholdMask(
+            for box in (amp.getRawBBox(), amp.getRawHorizontalOverscanBBox(),\
+                        amp.getRawVerticalOverscanBBox()):
+                dataView = maskedImage.Factory(maskedImage, box)
+                isr.makeThresholdMask(
                 maskedImage = dataView,
```

Comment:
I'm saying that the maskedImage line should be indented relative to the `isr.makeThresholdMask` above it. Without that it's hard to tell at first glance that the `maskedImage` line contains arguments for the previous line. Previously there was the correct indentation but the `for` loop moved the `isr.makeThresholdMask` in without adjusting the arguments to it.